app/assets/stylesheets/application.scss

```diff
@@ -1,2 +1,6 @@
 @import 'bootstrap';
 
+img{
```

Comment:
I would personally give this a more specific name. logo-img or header-img or something